crates/bevy_ecs/src/lib.rs

```diff
@@ -45,7 +45,7 @@ pub use bevy_ptr as ptr;
 ///
 /// This includes the most common types in this crate, re-exported for your convenience.
 pub mod prelude {
-    #[allow(deprecated)]
+    #[expect(deprecated)]
```

Comment:
Is this meant to remain?

Reply:
Yea `apply_deferred` is still deprecated.